apisix/schema_def.lua

```diff
@@ -688,7 +688,9 @@ _M.global_rule = {
     type = "object",
     properties = {
         id = id_schema,
-        plugins = plugins_schema
+        plugins = plugins_schema,
+        create_time = timestamp_def,
+        update_time = timestamp_def
```

Comment:
Need to inject the time when creating global rule, like this: https://github.com/apache/apisix/blob/7c5df5d343ef79f9a292a8eb621fec72280d13ea/apisix/admin/routes.lua#L200
Also need to update the documentation.

Reply:
@spacewander oh, good catch. I forgot that the Admin API is still in use and APISIX can also write the two fields into etcd. I would re-new this PR soon, as well as the docs and more test cases. Thank you very much.

Reply:
@spacewander PR renewed, with add missing time injection as well as relevant test cases. Regarding to doc, sorry for not getting where I can add, the https://github.com/apache/apisix/blob/master/doc/architecture-design.md#global-rule seems no need to mentioned that. Thanks.

Reply:
One more thing, I removed some test cases since they seems only for testing no `create_time` and `update_time`. Please let me know if I was wrong. Thanks.